android/src/main/kotlin/com/sap/gigya_flutter_plugin/GigyaSDKWrapper.kt

```diff
@@ -723,6 +725,10 @@ class GigyaSDKWrapper<T : GigyaAccount>(application: Application, accountObj: Cl
             arguments["parameters"] as MutableMap<String, Any>?
                 ?: mutableMapOf()
 
+        (arguments["isDebug"] as? Boolean)?.let { isDebug ->
+            if (isDebug) WebView.setWebContentsDebuggingEnabled(true).also { Log.d("SCREENSET_TAG", "Screenset Debuggable") }
```

Comment:
Do we really need the `Log.d()` here?

Reply:
I have removed all the log functions from the PR